lib/synapse/service_watcher/zookeeper.rb

```diff
@@ -123,81 +128,89 @@ def create(path)
 
     # find the current backends at the discovery path
     def discover
-      log.info "synapse: discovering backends for service #{@name}"
-
-      new_backends = []
-      @zk.children(@discovery['path'], :watch => true).each do |id|
-        begin
-          node = @zk.get("#{@discovery['path']}/#{id}")
-        rescue ZK::Exceptions::NoNode => e
-          # This can happen when the registry unregisters a service node between
-          # the call to @zk.children and @zk.get(path). ZK does not guarantee
-          # a read to ``get`` of a child returned by ``children`` will succeed
-          log.error("synapse: #{@discovery['path']}/#{id} disappeared before it could be read: #{e}")
-          next
+      statsd.time('synapse.watcher.zk.discover.elapsed_time', tags: ["zk_cluster:#{@zk_cluster}", "zk_path:#{@discovery['path']}"]) do
```

Comment:
can we add a counter metric here so that we know who (`@name`) gets update of a particular service (@discoery['path'])